features/support/hooks.rb

```diff
@@ -4,7 +4,12 @@
   $fail_on_js_error = true
 end
 
-After do
```

Comment:
Hmmm the two `After` blocks are a little confusing. I think a quick comment above each of them would aid the reader, e.g.:

```ruby
# Check for backend errors
After do |scenario|
  # ...

# Check for frontend errors
After do |scenario|
  # ...
```

Reply:
Sure, I've done this in https://github.com/alphagov/smokey/pull/931/commits/8f56cee879ac8d0bb1389bfb656cc794e87a25f7.